src/constants/queryKey.ts

```diff
@@ -1 +1,2 @@
+export const ROOM_UUID_INFO = 'roomUuidInfo';
 export const MY_INFO_KEY = 'myInfo';
```

Comment:
```suggestion
export const QUERY_KEY = {
  ROOM: {
    ROOM_UUID_INFO: 'roomUuidInfo',
  },
  MEMBERS: {
    MY_INFO_KEY: 'myInfo',
  },
};
```
쿼리 키도 API 기능 별로 위와 같이 객체로 묶어 놓으면 찾기 편할 것 같네요!
어제 오후에 수현님과 의논할 때 나온 부분인데 같이 의논해보고 어느 기준에서 적용해보면 좋을지 수영님 의견도 궁금합니다!

Reply:
위와 같이 변경하면 쿼리키를 사용할 때 `QUERY_KEY.ROOM.ROOM_UUID_INFO`와 같이 길어지는 단점은 있으나 자동완성해서 만들기가 편하다는 장점이 있습니다!

Reply:
객체 의견 좋다고 생각해요! 생각해본 적 없었는데 자동완성 시 편할 것 같습니다! 저는 그렇게 진행해보겠습니다~

Reply:
저도 관심사별로 객체로 관리하는 것 너무 좋은 것 같아요!!